### PR TITLE
People data for positive user

### PR DESCRIPTION
**Pull request by Josiah-Caldwell**
Acquire People contacts from a positive user
Merge People contacts and Calendar contacts

### CODE REVIEW

File: code/src/main/java/com/onlinecontacttracing/servlets/authentication/CheckForApiAuthorizationServlet.java [hunk 1]
```diff
@@ -1,143 +0,0 @@
-package com.onlinecontacttracing.authentication;
```
Comment (nvergel) [inline code/src/main/java/com/onlinecontacttracing/servlets/authentication/CheckForApiAuthorizationServlet.java]: Why was this servlet deleted? We needed it for the other servlets

File: code/src/main/java/com/onlinecontacttracing/servlets/get-data/PeopleDataForPositiveUser.java [hunk 2]
```diff
@@ -20,9 +53,40 @@ public void run() {
...
-    // Store data or replace old data with newer data.
-    // ofy.save().entity(positiveUserContacts).now();
+      // Store all connections available.
+      List<Person> connections = response.getConnections();
+      if (connections != null && connections.size() > 0) {
+        for (Person person : connections) {
+            List<Name> names = person.getNames();
+            if (names != null && names.size() > 0) {
+                positiveUserContacts.add(person.getNames().get(0).getDisplayName()
+                , person.getEmailAddresses().get(0).getDisplayName());
```
Comment (nvergel) [inline code/src/main/java/com/onlinecontacttracing/servlets/get-data/PeopleDataForPositiveUser.java]: nit: comma should go on previous line

File: code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java [hunk 3]
```diff
@@ -30,18 +30,24 @@ String getServletURIName() {
...
     peopleInfo.start();
     contactInfo.start();
     
     peopleInfo.join();
     contactInfo.join();
 
-    // TODO Load PositiveUserContacts from objectify
-    // TODO call mergeContactListsFromPeopleAPI(contactsFromPeople)
+    try {
+      PositiveUserContacts fullContacts = ofy.load().type(PositiveUserContacts.class).id(userId).now();
```
Comment (nvergel) [inline code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java]: lets not do a try catch, we want the application to proceed even if we dont use contacts api, just check that fullContacts is not null and if it is make a new object to load into objectify

File: code/src/main/java/com/onlinecontacttracing/storage/data-storage/PositiveUserContacts.java [hunk 4]
```diff
@@ -48,7 +48,11 @@ public void add(String name, String email) {
   }
 
   //TODO
-  public void mergeContactListsFromPeopleAPI(Set<PotentialContact> contactsFromPeople) {}
+  public void mergeContactListsFromPeopleAPI(Set<PotentialContact> contactsFromPeople) {
+      for(PotentialContact contact: contactsFromPeople) {
+          listOfContacts.add(contact.getName(), contact.getEmail());
```
Comment (nvergel) [inline code/src/main/java/com/onlinecontacttracing/storage/data-storage/PositiveUserContacts.java]: you can replace the for loop with .addAll(contactsFromPeople)

File: code/src/main/webapp/JavaScript/script.js [hunk 5]
```diff
@@ -179,17 +179,14 @@ function attachSignin(element, negativeUser) {
...
-    .then(url => window.location = url);
-
+    params.append('systemMessage', 'VERSION_1');
+    params.append('localityResource', 'US');
+    params.append('messageLanguage', 'SP');
+    console.log("here");
+    console.log(params);
+    fetch(new Request('/send-messages', {method: 'POST', body: params})).then(console.log("cool"));
+    // url => window.location = url
+    console.log("cool2");
```
Comment (nvergel) [inline code/src/main/webapp/JavaScript/script.js]: what's happening here?

File: code/src/main/webapp/WEB-INF/credentials.json [hunk 6]
```diff
@@ -0,0 +1 @@
+{"web":{"client_id":"1080865471187-u1vse3ccv9te949244t9rngma01r226m.apps.googleusercontent.com","project_id":"covid-catchers-fixed-gcp","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"LpbG3OQ9SqSckXfkWikxXFUn","redirect_uris":["https://covid-catchers-fixed-gcp.ue.r.appspot.com"],"javascript_origins":["https://8080-ac896ae1-5f0c-45b5-8dfe-19fa4d3d8699.us-east1.cloudshell.dev","https://covid-catchers-fixed-gcp.ue.r.appspot.com"]}}
```
Comment (nvergel) [inline code/src/main/webapp/WEB-INF/credentials.json]: remove this file

**Review by nvergel — CHANGES_REQUESTED** [review]

File: code/src/main/java/com/onlinecontacttracing/servlets/get-data/PeopleDataForPositiveUser.java [hunk 7]
```diff
@@ -20,9 +53,40 @@ public void run() {
...
+            } else {
+                System.out.println("No names available for connection.");
```
Comment (gradywardgoogle) [inline code/src/main/java/com/onlinecontacttracing/servlets/get-data/PeopleDataForPositiveUser.java]: Let's remove these statements - I think they're just for debugging, right? Same below.

File: code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java (hunk 3 above)
Comment (gradywardgoogle) [reply code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java]: +1

File: code/src/main/webapp/WEB-INF/credentials.json (hunk 6 above)
Comment (gradywardgoogle) [reply code/src/main/webapp/WEB-INF/credentials.json]: +1

File: code/src/main/webapp/JavaScript/script.js (hunk 5 above)
Comment (Josiah-Caldwell) [reply code/src/main/webapp/JavaScript/script.js]: Don't know I think this is from the send-msg-pull branch

**Review by gradywardgoogle — APPROVED** [review]

**Pull request merged**